perception/object_recognition/tracking/multi_object_tracker/src/tracker/model/tracker_base.cpp

```diff
@@ -20,17 +20,27 @@
 #include "multi_object_tracker/tracker/model/tracker_base.hpp"
 #include "multi_object_tracker/utils/utils.hpp"
 
-Tracker::Tracker(const ros::Time & time, const int type)
-: uuid_(unique_id::fromRandom()),
-  type_(type),
+#include <algorithm>
+#include <random>
+
+Tracker::Tracker(const rclcpp::Time & time, const int type)
+: type_(type),
   no_measurement_count_(0),
   total_measurement_count_(1),
   last_update_with_measurement_time_(time)
 {
+  // Generate random number
+  std::generate(uuid_.uuid.begin(), uuid_.uuid.end(), [] {
+    std::random_device rd;
```

Comment:
I think it is more idiomatic to initialize these outside `std::generate`, and a uniform distribution is fine too but I find `std::independent_bits_engine` a bit more concise:
```
std::mt19937 rng(std::random_device{}());
std::independent_bits_engine<std::mt19937, 8, uint8_t> byte_rng(rng);
std::generate(uuid_.uuid.begin(), uuid_.uuid.end(), byte_rng);
```

Reply:
Thanks for the suggestion, I don't know too much about this and was mostly following the way it was computed in the `unique_id` package though I guess this might/would change when that package is ported. I've taken your suggestion into account here: 0febc64